VMEncryption/main/check_util.py

```diff
@@ -168,10 +149,41 @@ def validate_volume_type(self, public_settings):
         if not volume_type.lower() in map(lambda x: x.lower(), supported_types) :
             raise Exception("Unknown Volume Type: {0}, has to be one of {1}".format(volume_type, supported_types))
 
+    def validate_lvm_os(self, public_settings):
```

Comment:
Please migrate the tests for the previous method into tests for this method (and maybe add a few cases)

Reply:
done